javascript/javascript1/week4/week04homework.js

```diff
@@ -0,0 +1,70 @@
+let listToDo =[];
```

Comment:
Here the key word can be const as you do not reassign to this variable, instead push in to it.

Reply:
I created a let because I was going to push the values and therefore indirectly I am updating the value of the variable. off-course not reassigning though.

Reply:
Yes, when you are pushing things to an array still you can use const.
Just remember: Use const until it doesn't let you :)